chapters/proposta-design/modello-caos/concetti/concetti.tex

```diff
@@ -18,27 +18,31 @@ \subsubsection{Resoconto del giorno a livello nazionale}
     \item nuovi decessi;
     \item nuovi ingressi in terapia intensiva;
     \item nuovi tamponi;
-    \item nuovi dimessi/guariti (in questa proprietà sono conteggiate tanto i soggetti guariti, ossia quelli risultati negativi al tampone molecolare, quanto coloro che sono stati dimessi dall'ospedale, nella consapevolezza che le loro migliorate condizioni di salute siano prossime alla guarigione);
+    \item nuovi dimessi/guariti:
+    \begin{itemize}
+        \item il guarito è colui il quale risolve i sintomi dell’infezione da Covid-19 e che risulta negativo in due test consecutivi, effettuati a distanza di 24 ore uno dall’altro, per la ricerca di SARS-CoV-2\footnote{La definizione è quella fornita dal Comitato tecnico scientifico nella comunicazione del 19 marzo 2020 e rinvenibile al seguente \href{http://www.salute.gov.it/portale/nuovocoronavirus/dettaglioNotizieNuovoCoronavirus.jsp?lingua=italiano&menu=notizie&p=dalministero&id=4274}{link}.}.
```

Comment:
Siamo sicuri? Ero rimasto che ora servisse un tampone e non più due. Chi se ne frega in realtà ai fini del progetto, ma è giusto per non dire cavolate... @MickPerl

Reply:
Così ho capito leggendo dal link